server/test/swagger_2/swagger_2.test.js

```diff
@@ -0,0 +1,99 @@
+const supertest = require('supertest');
+
+const app = require('../../app');
+const request = supertest(app);
+
+const jsonResponse1 = require('./data/output/jsonResponse1.json');
+const jsonResponse2 = require('./data/output/jsonResponse2.json');
+const { yamlResponse1, yamlResponse2 } = require('./data/output/yamlResponses');
+
+
+describe('POST /api/specification/swagger_2/openapi_3', () => {
+  //Swagger 2 (JSON) to OpenAPI 3 (JSON)
+  it('should convert swagger_2 (json) to openapi_3 (json) ', async () => {
+    const res = await request
```

Comment:
We should add a try-catch block if we're using `await`. It might be simpler to use callback design here though.

Reply:
@AbhiramH Changed it to a callback design.